src/main/java/it/rate/webapp/controllers/api/InterestRestController.java

```diff
@@ -30,7 +30,7 @@ public class InterestRestController {
   private final UserService userService;
   private final LikeService likeService;
 
-  @GetMapping("/suggestions")
+  @PostMapping("/suggestions")

```

Comment:
Should this really be POST mapping? 
https://www.restapitutorial.com/lessons/httpmethods.html